docs/resources/User.md

```diff
@@ -24,23 +24,24 @@ There are other rules and restrictions not shared here for the sake of spam and
+| Field              | Type      | Description                                                                                          | Required OAuth2 Scope |
+| ------------------ | --------- | ---------------------------------------------------------------------------------------------------- | --------------------- |
+| id                 | snowflake | the user's id                                                                                        | identify              |
+| username           | string    | the user's username, not unique across the platform                                                  | identify              |
+| discriminator      | string    | the user's 4-digit discord-tag                                                                       | identify              |
+| avatar             | ?string   | the user's [avatar hash](#DOCS_REFERENCE/image-formatting)                                           | identify              |
+| bot?               | boolean   | whether the user belongs to an OAuth2 application                                                    | identify              |
+| system?            | boolean   | whether the user is an Official Discord System user (part of the urgent message system)              | identify              |
+| mfa_enabled?       | boolean   | whether the user has two factor enabled on their account                                             | identify              |
+| banner?            | ?string   | the user's [banner hash](#DOCS_REFERENCE/image-formatting)                                           | identify              |
+| accent_color?      | ?integer  | the user's banner color encoded as an integer representation of hexadecimal color code               | identify              |
+| locale?            | string    | the user's chosen [language option](#DOCS_REFERENCE/locales)                                         | identify              |
+| verified?          | boolean   | whether the email on this account has been verified                                                  | email                 |
+| email?             | ?string   | the user's email                                                                                     | email                 |
+| flags?             | integer   | the [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                         | identify              |
+| premium_type?      | integer   | the [type of Nitro subscription](#DOCS_RESOURCES_USER/user-object-premium-types) on a user's account | identify              |
+| public_flags?      | integer   | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                  | identify              |
+| avatar_decoration? | ?string   | the user's [avatar decoration hash](#DOCS_REFERENCE/image-formatting)                                | identify              |
```

Comment:
In what circumstances is this optional?

Reply:
Partial data I think

Reply:
> In what circumstances is this optional?

I'd assume this is the same situation with banners where it's only returned via GW and /users. I'll check in a second

Reply:
> > In what circumstances is this optional?
> 
> I'd assume this is the same situation with banners where it's only returned via GW and /users. I'll check in a second

From what I've seen, avatar_decoration seems to be always available, even on partial users.

Reply:
That would seem to be the case yeah 
![image](https://user-images.githubusercontent.com/42438262/209402372-a5c291d5-5c18-48bb-8d1b-df67908526d8.png)

Reply:
The only user field that doesn't seem to have avatar_decoration is Widget.members
![image](https://user-images.githubusercontent.com/67214928/209403004-cebacfdb-df81-4adf-b5f8-6646a7887516.png)

Reply:
That makes sense, I *guess*?

Reply:
it's present on normal user objects but not on bare user objects like webhook message authors